Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(
+    name: "RIBs",
+    platforms: [
+        .iOS(.v8),
+    ],
+    products: [
+        .library(name: "RIBs", targets: ["RIBs"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift", from: "5.0.1"),
+    ],
+    targets: [
+        .target(
+            name: "RIBs",
+            dependencies: ["RxSwift", "RxRelay"],
+            path: "ios/RIBs"
+        ),
+        .testTarget(
+            name: "RIBsTests",
+            dependencies: ["RIBs"],
+            path: "ios/RIBsTests",
+            exclude: ["ComponentizedBuilderTests.swift", "MultiStageComponentizedBuilderTests.swift"]
```

Comment:
Why did you exclude these two?

Reply:
Unfortunately, I don't remember. I've added them back and seems like everything builds well.